build.ps1

```diff
@@ -223,6 +227,7 @@ if (!(Test-Path $CAKE_EXE)) {
 # Build Cake arguments
 $cakeArguments = @("$Script");
 if ($Target) { $cakeArguments += "-target=$Target" }
+if ($ToolVersion) { $cakeArguments += "-ToolVersion=$ToolVersion" }
```

Comment:
I ever tried this, but not work. The correct way is add this parameter into the scriptArgs.